app/views/application/equality-monitoring/ethnic-background.html

```diff
@@ -67,25 +33,27 @@
         }
       },
       items: [{
+        text: "Indian"
+      },
+      {
+        text: "Pakistani"
+      },
+      {
```

Comment:
Updated to follow the order of the [Equality information pattern](https://design-system.service.gov.uk/patterns/equality-information/) which I think in turn is from the ONS pattern/census. Rather than alphabetical I think it’s in rough order of population size (in the UK).

This does introduce a risk of perceived bias though.

Reply:
Ok, sounds good. I note the new order is also what the Census 2020 used.